Week1/exercise_2.js

```diff
@@ -0,0 +1,44 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'world'
+});
+
+/*Connect to SQL server*/
+connection.connect(err => {
+    if (err) {
+        return console.error('Connection error: ' + err.stack);
```

Comment:
Same comment as this: https://github.com/HackYourFuture/databases/pull/567/files#r1736012412